x-pack/plugins/cases/public/components/create/flyout/create_case_flyout.tsx

```diff
@@ -47,64 +43,50 @@ const GlobalStyle = createGlobalStyle<{ theme: { eui: { euiZLevel5: number } } }
 
 // Adding bottom padding because timeline's
 // bottom bar gonna hide the submit button.
+// might not need for obs, test this when implementing this component
 const StyledEuiFlyoutBody = styled(EuiFlyoutBody)`
   ${({ theme }) => `
-    && .euiFlyoutBody__overflow {
-      overflow-y: auto;
-      overflow-x: hidden;
-    }
-
-    && .euiFlyoutBody__overflowContent {
-      display: block;
-      padding: ${theme.eui.paddingSizes.l} ${theme.eui.paddingSizes.l} 70px;
-      height: auto;
-    }
-  `}
+      && .euiFlyoutBody__overflow {
+        overflow-y: auto;
+        overflow-x: hidden;
+      }
+  
+      && .euiFlyoutBody__overflowContent {
+        display: block;
+        padding: ${theme.eui.paddingSizes.l} ${theme.eui.paddingSizes.l} 70px;
+        height: auto;
+      }
+    `}
 `;
 
 const FormWrapper = styled.div`
   width: 100%;
 `;
 
-const CreateCaseFlyoutComponent: React.FC<CreateCaseModalProps> = ({
-  afterCaseCreated,
-  onCloseFlyout,
-  onSuccess,
-  owner,
-  disableAlerts,
-}) => {
-  const { cases } = useKibana<TimelinesStartServices>().services;
-  const createCaseProps = useMemo(() => {
-    return {
-      afterCaseCreated,
-      onCancel: onCloseFlyout,
-      onSuccess,
-      withSteps: false,
-      owner: [owner],
-      disableAlerts,
-    };
-  }, [afterCaseCreated, onCloseFlyout, onSuccess, owner, disableAlerts]);
-  return (
+export const CreateCaseFlyout = React.memo<CreateCaseFlyoutProps>(
+  ({ afterCaseCreated, onClose, onSuccess }) => (
     <>
       <GlobalStyle />
-      <StyledFlyout
-        onClose={onCloseFlyout}
-        data-test-subj="create-case-flyout"
-        maskProps={{ className: maskOverlayClassName }}
-      >
+      <StyledFlyout onClose={onClose} data-test-subj="create-case-flyout">
         <EuiFlyoutHeader hasBorder>
           <EuiTitle size="m">
             <h2>{i18n.CREATE_TITLE}</h2>
           </EuiTitle>
         </EuiFlyoutHeader>
         <StyledEuiFlyoutBody>
-          <FormWrapper>{cases.getCreateCase(createCaseProps)}</FormWrapper>
+          <FormWrapper>
+            <CreateCase
+              afterCaseCreated={afterCaseCreated}
+              onCancel={onClose}
+              onSuccess={onSuccess}
+              withSteps={false}
+              disableAlerts={true}
```

Comment:
In the main app page is where security (disableAlerts: false) differs from obs (disableAlerts: true), you are right in that case. 
In the flyout is always passed `true` from both plugins, it would not be needed, but I can keep it anyway, do you think it would be more consistent?

Reply:
We discussed it offline and the flyout will receive the `disableAlerts` as props.